packages/eslint-plugin/docs/rules/import-order.md

```diff
@@ -0,0 +1,49 @@
+# Enforce consistent import ordering (import-order)
+
+## Rule Details
+
+This rule enforces a consistent import ordering that matches the TypeScript command `Organise Imports`.
+
+Examples of **incorrect** code for this rule with the default options:
+
+<!-- prettier-ignore -->
+```js
+/*eslint @typescript-eslint/import-order: "error"*/
+
+import { b, a } from 'foo'
+```
+
+<!-- prettier-ignore -->
+```js
+/*eslint @typescript-eslint/import-order: "error"*/
+
+import a from 'b';
+import b from 'a';
+```
+
+Examples of **correct** code for this rule with the default options:
+
+<!-- prettier-ignore -->
+```js
+/*eslint @typescript-eslint/import-order: "error"*/
+
+import a from 'a';
+import { b, c } from 'bc';
+import d from 'd';
+```
+
+<!-- prettier-ignore -->
+```js
+/*eslint @typescript-eslint/import-order: "error"*/
+
+import b from 'a';
+import a from 'b';
```

Comment:
Can you add example (and tests maybe) with relative imports ?

import c from './c'; // same folder
import d from '../d'; // another folder